picoquic/quicctx.c

```diff
@@ -330,6 +330,16 @@ void picoquic_registered_token_clear(picoquic_quic_t* quic, uint64_t expiry_time
     } while (!end_reached);
 }
 
+void picoquic_adjust_max_connections(picoquic_quic_t * quic, uint32_t max_nb_connections)
+{
+    quic->max_number_connections = max_nb_connections;
+}
```

Comment:
That's probably not sufficient. During the initial context creation, "max_nb connections" is used to set the dimension of the hash tables. If we increase the hash number without resizing the hash tables, the system might become slow.

Reply:
oh very good point about technically being able to raise that number higher. we could store another variable called "working maximum" something like that. and restrict it to never raising higher than the maximum set in `picoquic_create`.